core/src/actions/navigation.ts

```diff
@@ -192,15 +193,42 @@ export class NavigationList extends OptionallyAuthenticatedAction {
     }
 
     if (navigationMode === "config") {
-      navigationItems = [
-        { type: "link", title: "Apps", href: "/apps" },
-        { type: "link", title: "Sources", href: "/sources" },
-        { type: "link", title: "Properties", href: "/properties" },
-        { type: "link", title: "Groups", href: "/groups" },
-        { type: "link", title: "Destinations", href: "/destinations" },
-        { type: "link", title: "Plugins", href: "/plugins" },
-        { type: "link", title: "Validate", href: "/validate" },
-      ];
+      const user = await ConfigUser.get();
+      if (user) {
+        navigationItems = [
+          {
+            type: "link",
+            title: "Apps",
+            href: "/apps",
```

Comment:
Per your comment above, my thoughts are [links](https://fontawesome.com/v5.15/icons/link?style=solid), since they're things you can connect, or [squares](https://fontawesome.com/v5.15/icons/th-large?style=solid), since it somewhat mirrors the imagery we have on our site and on the apps page in the ui.  My other thought was a cube since they're the building blocks, but fontawesome's cube looks like a company logo to me.

Reply:
+1 for the squares. I've seen this be used for "apps" in other... _apps_... too.

Reply:
@tealjulia @pedroslopez @evantahler Any idea what I have to do to get this working? I can't get anything other than what we're already currently using to work, even though the appropriate files are clearly there. Are we enabling a specific set somehow?

Reply:
Did you add it to the [icon library file](https://github.com/grouparoo/grouparoo/blob/7687269d90e4fad52374d63fb2eeb8c068e8c698/ui/ui-components/components/icons.ts)?

Reply:
omg you are my hero